### PR TITLE
Adding the new Selenium TNG

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +86,12 @@ <h2>Narrative documentation</h2>
    <li><a href=intro.html#types_of_testing>Types of testing</a>
    <li><a href=intro.html#about_this_documentation>About this documentation</a>
   </ul></li>
- <li><a href=install.html#selenium_installation>Selenium installation</a>
+ <li><a href=side.html#side>Selenium-IDE</a>
```
Comment (shs96c) [inline index.html:89]: Nit: It's `Selenium IDE` (without a dash)

File: side.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<meta charset=utf-8>
+<title>Introduction</title>
+<link rel="icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel=stylesheet href=se.css>
+<link rel=prev href=quick.html title="Quick Tour">
+<link rel=next href=install.html title=Install>
+<script src=docs.js></script>
+
+<h1>Selenium-IDE</h1>
```
Comment (shs96c) [inline side.html:11]: Same nit,

File: side.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<h3>Introduction</h3>
+
+<p>The <em>Selenium-IDE(Integrated Development Environment) </em>is the tool you use to develop your
```
Comment (shs96c) [inline side.html:15]: :)

Comment (shs96c) [inline side.html:15]: `Environment)</em> is`

Comment (shs96c) [inline side.html:15]: You don't develop tests with IDE (Noooooooo!) It's a record/playback tool that is designed to offer the selenium API and to allow people to export to a programming language.

File: side.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    Selenium test cases. It’s an easy-to-use Chrome and Firefox extension and is generally the most
```
Comment (shs96c) [inline side.html:16]: It's an easy-to-use Web Extension, usable in Chrome, Firefox, and any browser that supports that standard.

File: side.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    efficient way to develop test cases. It records the users actions in the browser for you, using
```
Comment (shs96c) [inline side.html:17]: Most efficient way? Eeek! It's great for quickly recording tests, fantastic for folks who can't code, and a nice way to bootstrap those first tests on a new extant site, but it's _far_ from being the most efficient way to develop tests.

**Review by shs96c — COMMENTED** [review]
I can see the positive approach being taken here, and understand the reasoning, but as the official selenium docs it may be better to position IDE as a starter tool, something for non-programmers to use, or a tool for throwing together a test case on an extant site.

I'll be honest, I'm not sure of the best way of phrasing this in the docs. :)

**Comment by manoj9788** [comment]
@shs96c Agreed and will change it to sound better!

File: side.html (hunk 5 above)
Comment (titusfortner) [reply side.html:17]: Can we back off of the usage of "IDE is for Test Cases" in general? It is a browser automation record and playback tool that can output Selenium code. This language makes it sound like we want people to create their test suites with it, and we should be discouraging people from doing that.

Comment (manoj9788) [reply side.html:17]: Sure, Titus.

**Pull request merged**